library/src/wu/seal/jsontokotlin/JsonToKotlinBuilder.kt

```diff
@@ -0,0 +1,209 @@
+package wu.seal.jsontokotlin
+
+import extensions.chen.biao.KeepAnnotationSupport
+import extensions.jose.han.ParcelableAnnotationSupport
+import extensions.ted.zeng.PropertyAnnotationLineSupport
+import extensions.wu.seal.*
+import extensions.xu.rui.PrimitiveTypeNonNullableSupport
+import wu.seal.jsontokotlin.test.TestConfig
+
+
+/**
+ * To get Kotlin class code from JSON
+ */
+class JsonToKotlinBuilder() {
+
+    init {
+        TestConfig.apply {
+
+            isTestModel = true
+            isCommentOff = false
+            isOrderByAlphabetical = true
+            isPropertiesVar = false
+            targetJsonConvertLib = TargetJsonConverter.NoneWithCamelCase
+            propertyTypeStrategy = PropertyTypeStrategy.AutoDeterMineNullableOrNot
+            defaultValueStrategy = DefaultValueStrategy.None
+            isNestedClassModel = true
+            customPropertyAnnotationFormatString = "@Optional\n@SerialName(\"%s\")"
```

Comment:
If you don't choose `TargetJsonConverter.Custom`, There is no need to give value to customPropertyAnnotation*

Reply:
I just copied it from the `TestConfig.kt` 

https://github.com/wuseal/JsonToKotlinClass/blob/51245c785208e421dd199c6fb8141ca9510f6135/src/main/kotlin/wu/seal/jsontokotlin/test/TestConfig.kt#L29

Reply:
@theapache64 Got it, Although there is no problem did as this

Reply:
Yes! Remove it.

Reply:
Also, where are the methods about setting these properties: `customPropertyAnnotationFormatString` `customAnnotaionImportClassString` `customClassAnnotationFormatString`. they are need when set TargetJsonConverter to `Custom`

Reply:
Ok. Will add